### PR TITLE
The book is available for pre-order!

### PR DESCRIPTION
**Pull request by shiffman**
https://nostarch.com/nature-code

### CODE REVIEW

**Comment by shiffman** [comment]
@jasongao97 I'm not sure the proper way to link the "button" let me know what you think!

**Comment by jasongao97** [comment]
I believe they suggest using `<a/>` tag for external links, and I've made the change.

File: src/components/Header.js [hunk 1]
```diff
@@ -16,9 +16,12 @@ const Header = () => {
           </h2>
         </div>
 
-        <button className="hidden sm:block bg-white hover:bg-gray-100 text-noc-600 rounded font-semibold px-3 py-1 shadow">
+        <a
+          href="https://nostarch.com/nature-code"
+          className="hidden sm:block bg-white hover:bg-gray-100 text-noc-600 rounded font-semibold px-3 py-1 shadow"
+        >
           Buy this book
-        </button>
```
Comment (shiffman) [inline src/components/Header.js:21]: Should the button tag still be closed?

**Comment by jasongao97** [comment]
The link is working. Can we merge this?

**Review by jasongao97 — APPROVED** [review]

**Comment by shiffman** [comment]
Yes! I just wasn't sure if there was a missing close button tab? https://github.com/nature-of-code/noc-book-2023/pull/389/files#r1295839709

**Comment by shiffman** [comment]
Great! Never mind about my question :)

**Pull request merged**